shop/src/themes/shared/checkout/_Summary.js

```diff
@@ -87,19 +86,29 @@ export const OrderSummary = ({ cart, discount = true }) => {
   )
 }
 
-const Row = ({ title, quantity, img, price }) => (
+const Row = ({
+  title,
+  productOptions,
+  quantity,
+  img,
+  price,
+  currencyOptions
+}) => (
   <>
-    <div className="border-b pb-3 dark:border-gray-700">
-      <div className="flex items-center text-sm font-semibold">
-        <img className="h-16 mr-5" src={img} />
-        {title}
+    <div className="flex justify-between pb-3 dark:border-gray-700">
+      <div className="flex relative items-center text-sm font-semibold">
+        <img className="h-16 w-16 box-content object-scale-down" src={img} />
+        <span className="absolute -top-3 -right-3 h-6 w-6 rounded-full bg-gray-600 text-sm text-white flex justify-around items-center dark:border-gray-700">
+          {quantity}
+        </span>
+      </div>
+      <div className="flex flex-col justify-center">
+        <div className="font-semibold">{title}</div>
+        <div className="text-xs font-light">{productOptions}</div>
+      </div>
+      <div className="flex items-center justify-end font-semi-bold dark:border-gray-700">
+        {formatPrice(quantity * price, currencyOptions)}
```

Comment:
Reason for changing the props passed to `Row`:

When a user checks out, they are first shown the page where they can modify their cart if need be (Example: https://user-images.githubusercontent.com/10854442/143019839-698416b0-07d2-452d-84d0-af5981f02f4f.png). The price displayed on each row here is calculated -- `quantity * (price per item)`.
On the next screen, when they're asked to input their contact and shipping details, we display an **Order Summary** on the right, but do not format the price in the same way as before. 

Reference: https://github.com/OriginProtocol/dshop/blob/f65491f003fa96271f9b6f7c6554d05ae039e737/shop/src/themes/shared/Cart.js#L45-L51